packages/react-reconciler/src/__tests__/ReactSuspenseList-test.js

```diff
@@ -2443,41 +2443,30 @@ describe('ReactSuspenseList', () => {
 
     expect(ReactNoop).toMatchRenderedOutput(null);
 
-    ReactNoop.act(() => {
+    await ReactNoop.act(async () => {
```

Comment:
I rewrote this test to decouple it from the weird current behavior of discrete updates (async for a bit, unless another event  comes in). Passes in both implementations.